content/src/Server.ts

```diff
@@ -73,6 +73,7 @@ export class Server {
     this.registerRoute('/available-content', controller, controller.getAvailableContent)
     this.registerRoute('/audit/:type/:entityId', controller, controller.getAudit)
     this.registerRoute('/deployments', controller, controller.getDeployments)
+    this.registerRoute('/contents/:hashId/active-entity', controller, controller.getActiveDeploymentByContentHash)
```

Comment:
There could be more than one active entity that references a hash. We should be returning a list here

Reply:
Fixed!

Reply:
Should we change the endpoint so that it's `/contents/:hashId/active-entities`?